spanner/google/cloud/spanner/__init__.py

```diff
@@ -18,6 +18,13 @@
 import pkg_resources
 __version__ = pkg_resources.get_distribution('google-cloud-spanner').version
 
+from google.cloud.spanner._helpers import BOOL_PARAM_TYPE
```

Comment:
Seems like a perfectly reasonable use of https://en.wikipedia.org/wiki/Facade_pattern to me:  the "implementation" is private, but the things themselves are exposed to the public.